.github/workflows/backup.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: Create external backup
+
+# yamllint disable-line rule:truthy
+on:
+  schedule:
+    # Run every day at 10am
+    - cron: '0 10 * * *'
```

Comment:
Databázi lokálně zálohujeme v pět ráno (když selže, tak to zkoušíme znovu vždy po hodině znovu).

Navrhuju tohle pustit v v půl šesté a pak jako zálohu nechat třeba půl jedenáctou (protože CRON nedovoluje co vím zápis dvou časů s různými minutami). Abychom měli v případě katastrofy co nejčerstvější zálohu. Když paranoia, tak pořádná.

`cron: '30 06,10 * * *'`

Reply:
Původně jsem měl v šest, ale pro jistotu jsem to posunul na desátou, kdy si myslím, že ta záloha už by tam mohla být i kdyby se náhodou zrovna nesešlo časové pásmo (tohle je totiž tak trochu geo-replikace). Můžeme to klidně posunout, nemyslím, že by to bylo důležité, pokud to bude chtít selhat, selže to minutu před spuštěním. Ale rozhodně to nemůže bez dalších úprav běžet víc jak jednou denně (jednak je jméno vzdáleného archivu je bez času a druhak dělat dva snapshoty se stejným obsahem je spíš kontraproduktivní).

Pokud chceš být paranoidní, tak si místo spouštění cronem naimplementuj webhook po vyrobení zálohy na serveru, pak to můžeš mít téměř instantní.

Reply:
> Pokud chceš být paranoidní, tak si místo spouštění cronem naimplementuj webhook po vyrobení zálohy na serveru, pak to můžeš mít téměř instantní.

Celá tahle akce vyřeší 99.9 % problému a zbytek je moje honba za dokonalostí. Takže si to nechám na jindy.